granular-access-controlled-logging-app/README.md

```diff
@@ -0,0 +1,48 @@
+# granular-access-controlled-logging-app (better naming is welcomed)
+
+This is a sample application of logging app that takes advantage of CCF's ability for granular access control.
+
+## Use case
+
+You need to build a logging system that can be checked by auditors.
+The amount of data exposed should be minimized.
```

Comment:
"Data logged to the system must only be revealed to authorised users, the service operator must not be able to access the data."